src/App.js

```diff
@@ -22,27 +22,26 @@ function App() {
     setVisibleMentorIds(searchResults.map((result) => result.item.id));
   };
 
-  const [isModalActive, setIsModalActive] = useState(false);
+  const [isModalOpen, setIsModalOpen] = useState(false);
   const [activeMentorId, setActiveMentorId] = useState(null);
 
   const readMoreHandler = (mentorId) => {
-    setIsModalActive(true);
+    setIsModalOpen(true);
     setActiveMentorId(mentorId);
   };
 
   const closeHandler = () => {
-    setIsModalActive(false);
+    setIsModalOpen(false);
     setActiveMentorId(null);
   };
 
   return (
     <div className="container">
       <Header onSearch={searchHandler} />
 
-      {isModalActive && (
+      {isModalOpen && (
```

Comment:
Also, this check probably isn’t needed anymore.

Reply:
This check is still required unfortunately, as `ProfileModal` depends on `mentor` being set to an actual mentor.

We could make modal persistent even when closed, so that it takes on the previous mentor, but I think there's little point to making that change.

Reply:
Oh I see, didn’t catch that.